### PR TITLE
Fix Issue #1450

### PR DESCRIPTION
**Pull request by ahmedofali**
Allow Uploading multiple files under the same name using array name .  ex: "file.image_name"

### CODE REVIEW

**Comment by ahmedofali** [comment]
@brendt Can you please review this pull request thanks in advance.

File: src/FileAdder/FileAdderFactory.php [hunk 1]
```diff
@@ -29,7 +29,13 @@ public static function createFromRequest(Model $subject, string $key): FileAdder
     public static function createMultipleFromRequest(Model $subject, array $keys = []): Collection
     {
         return collect($keys)
+
```
Comment (brendt) [inline src/FileAdder/FileAdderFactory.php]: Remove this whitespace

**Comment by ahmedofali** [comment]
@brendt  I wrote tests please review and let me know.

**Comment by brendt** [comment]
Tagged as 7.6.2: https://github.com/spatie/laravel-medialibrary/releases/tag/7.6.2

**Pull request merged**